contribute-to-timescaledb/contribute-to-docs.md

```diff
@@ -0,0 +1,266 @@
+reading workload by using:
+*   Short sentences
+*   Plain language
+*   Common vocabulary
+*   A minimum number of eye-catching details such as admonitions
+
+For example, here's the same idea expressed three different ways, with varying
+clarity:
+
+|Style|Example|
+|-|-|
+|❌ Natural English|In order to perform X installation process, please ensure that all of the following steps are done ...|
+|❌ Tech writer's English|To perform the X installation process, verify you have done the following steps ...|
+|✅ Plain English|To install X, do these steps ...|
+
+### Tense and perspective
+Use simple present tense and the second person singular ("you"). Avoid the first
+person plural ("we," "us"), because it isn't clear who is included in that
+group.
+
+### Voice
+We prefer active voice, but don't be afraid to use passive when it serves
+better. For example, the most important words come first in an English sentence.
+If the object of an action is the most important part of a sentence, put it
+first, even if it means using passive.
+
+### Grammar
+Grammar rules matter only because everyone follows them. They change over time
+as public use changes.
+
+Don't worry about hard and fast rules, such as avoiding dangling participles,
+not splitting infinitives, and not ending sentences with prepositions. 
+
+Use grammar rules when they make the sentence clearer. For example, use commas
+to avoid ambiguous meanings.
+
+When grammar rules hurt more than help, ignore them.
+
+### User interface actions
+Refer to user interface actions like this:
+*   `Click` a button in a graphical user interface. Do not `Click on`.
+*   `Press` a key or key combination on a keyboard.
+*   `Type` words or numbers using a keyboard.
+*   `Check` or `uncheck` a checkbox.
+*   `Select` or `deselect` an item in a menu.
+*   `Navigate` to a page or location in a graphical user interface.
+
+### Prose linting with Vale
+The Timescale documentation uses the Vale prose linter to check content for
+language conventions. After you submit a pull request, a GitHub Action
+automatically runs Vale. It shows the results within the `Files changed` view of
+your pull request.
+
+Vale has three levels of admonitions:
+*   Errors. These block merging of your pull request into the official published
+    docs. They must be fixed before your contribution can be accepted.
+*   Warnings. Less serious than errors. Adopt these recommendations unless you
+    have a strong reason not to.
+*   Suggestions. The least serious level. Consider and adopt these
+    recommendations, unless your content is a special case or exception.
+
+## Structure
+
+### Information when it's needed, where it's needed
+Readers don't read documentation from cover to cover, like a book. They skim and
+skip. Don't assume that they know something because it's explained earlier in
+the document. Use cross-references to guide them to further information when
+they need it.  
+
```

Comment:
Perhaps add a reference to Every Page is Page One here? Or is that going too much into the content strategy weeds?